src/components/settings/EditProfile.tsx

```diff
@@ -0,0 +1,263 @@
+import { type Dispatch, type SetStateAction, useState } from "react";
+import { z } from "zod";
+import {
+  Button,
+  TextField,
+  Autocomplete,
+  createFilterOptions,
+  FormControl,
+} from "@mui/material";
+import EditIcon from "@mui/icons-material/Edit";
+import SaveIcon from "@mui/icons-material/Save";
+import MultipleSelectChip from "./MulitpleSelectChip";
+import universities from "./us_universities.json";
+import industries from "./industries.json";
+
+interface Profile {
+  id: number;
+  userId: string;
+  preferredName: string;
+  phoneNumber: string;
+  bio: string;
+}
+
+interface Account {
+  id: string;
+  userId: string;
+  type: string;
+  provider: string;
+}
+
+export interface Data {
```

Comment:
This is definitely available from tRPC router output types, as we are grabbing it straight from there. Perhaps the other interfaces in this file can be imported/inferred as well.